static/app/views/performance/newTraceDetails/traceDrawer/details/durationComparison.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from '@emotion/styled';
+
+import {Row} from 'sentry/components/events/interfaces/spans/newTraceDetailsSpanDetails';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {getDuration} from 'sentry/utils/formatters';
+
+type Props = {
+  duration: number;
+  title: string;
+  avgDuration?: number;
+  toolTipText?: string;
+};
+
+export const COLORS = {
+  faster: {
+    light: 'green100',
+    normal: 'green300',
+  },
+  slower: {
+    light: 'red100',
+    normal: 'red300',
+  },
+  equal: {
+    light: 'gray100',
+    normal: 'gray300',
+  },
+};
+
+function DurationComparison(props: Props) {
+  if (!props.avgDuration) {
+    return (
+      <Row title={props.title} toolTipText={props.toolTipText}>
+        <Duration>{getDuration(props.duration, 2, true)}</Duration>
+      </Row>
+    );
+  }
+
+  const delta = props.duration - props.avgDuration;
+  const deltaPct = `${Number(Math.abs((delta / props.avgDuration) * 100).toFixed(2))}%`;
+  const formattedAvgDuration = getDuration(props.avgDuration, 2, true);
+  const status = delta > 0 ? 'slower' : delta < 0 ? 'faster' : 'equal';
+
+  const deltaText =
+    status === 'equal'
+      ? t(`Equal to avg %s`, deltaPct, formattedAvgDuration)
```

Comment:
If it equals, we just shouldnt display anything. In practice though, this will rarely be the case as we have ms precision. I would probably add a 10% +/- margin and not display the avg if the number is close to avg (unless our confidence is super high, but I dont think thats the case). 

We need to be careful where and when we steal users attention. If we do it too often, we risk that they'll get used to seeing low % changes highlighted in red and stop paying attention to this part of the UI as in most cases, the numbers wont have a strong significance or be a strong signal that something is wrong